donkeycar/parts/oak_d_camera.py

```diff
@@ -0,0 +1,359 @@
+import logging
+import time
+from collections import deque
+import numpy as np
+try:
+    import depthai as dai
+except ImportError:
+    raise ImportError("Please install the depthai library. See https://docs.luxonis.com/en/latest/pages/tutorials/first_steps/#installing-depthai")
+
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
```

Comment:
Please remove - logging should not be done on a file / module basis but is passed through in manage.py from the environment or command line instead.